apps/web/client/src/app/api/chat/route.ts

```
@@ -84,7 +84,21 @@ export const getSupabaseUser = async (request: NextRequest) => {
     const { model, providerOptions } = await initModel(LLMProvider.ANTHROPIC, CLAUDE_MODELS.SONNET_4);
-    const systemPrompt = chatType === ChatType.CREATE ? getCreatePageSystemPrompt() : getSystemPrompt();
+    
+    let systemPrompt: string;
+    switch (chatType) {
+        case ChatType.CREATE:
+            systemPrompt = getCreatePageSystemPrompt();
+            break;
+        case ChatType.ASK:
+            systemPrompt = getAskModeSystemPrompt();
+            break;
+        case ChatType.EDIT:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)